examples/nrf24l01_btle/nrf24l01_btle.c

```diff
@@ -219,16 +219,16 @@ int nrf24_cfg(int fd)
   int32_t txpow = DEFAULT_TXPOWER;
 
   /**************************************************************************
-   * if using RATE_1Mbps from include/nuttx/wireless/nrf24l01.h,
+   * if using RATE_1MBPS from include/nuttx/wireless/nrf24l01.h,
    * tools/checkpatch.sh report error: Mixed case identifier found.
    *
    **************************************************************************/
 
-  nrf24l01_datarate_t datarate = 0; /* RATE_1Mbps */
+  nrf24l01_datarate_t datarate = 0; /* RATE_1MBPS */
```

Comment:
@xiaoxiang781216 do you know why `RATE_1MBPS` and `DELAY_1000US` are not used here, but `0` and `3` are used instead?